src/Request/FetchPickupPointsRequest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\PocztaPolskaBundle\Request;
+
+use Answear\PocztaPolskaBundle\DTO\PickupPoint;
+use Answear\PocztaPolskaBundle\Factory\PickupPointFactory;
+use Answear\PocztaPolskaBundle\Service\ConfigProvider;
+use ZipArchive;
+
+class FetchPickupPointsRequest
+{
+    private const ZIP_URL = 'https://odbiorwpunkcie.poczta-polska.pl/pliki.php?t=xmlK48S';
```

Comment:
This will be configurable, right?

Reply:
Its not needed, url doesnt change

Reply:
Are you sure? `t` parameter seems like some id of current content